client/components/data/query-jetpack-user-connection/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingJetpackUserConnection } from 'state/selectors';
+import { requestJetpackUserConnectionData } from 'state/jetpack/connection/actions';
+
+class QueryJetpackUserConnection extends Component {
+	static propTypes = {
+		siteId: PropTypes.number,
+		// Connected props
+		isRequesting: PropTypes.bool,
+		requestJetpackUserConnectionData: PropTypes.func.isRequired,
+	}
+
+	componentDidMount() {
+		this.request( this.props );
```

Comment:
We're already doing that, see https://github.com/Automattic/wp-calypso/pull/15599/files#diff-6a463ab01bf85f9ede30bd7a4663e845R33